internal/postprocessor/config.yaml

```diff
@@ -124,3 +124,435 @@ modules:
   - webrisk
   - websecurityscanner
   - workflows
+
+service-configs:
```

Comment:
Would it be possible to sort these alphabetically?

Reply:
They are, by input directory

Reply:
Regarding sort order, see comment below on `workstations`.

Reply:
Can modules now be derived from service-configs?

Reply:
They can not, a module can have N clients at different levels and modules don't have to be directly off the root.